### PR TITLE
Alibaba Cloud multiple tidb clusters management support

### PR DESCRIPTION
**Pull request by aylei**
### What problem does this PR solve? 
Add multiple tidb clusters management support in Alibaba Cloud 

### What is changed and how does it work?
Some reusable sub modules are created just like aws #650 , the scripts in `deploy/aliyun`  are changed accordingly to compose one kubernetes cluster with one or more tidb clusters.

Also, the `README` and `README-CN` and removed and the users are redirected to `docs-cn` and `docs` in favor of consistency.

This PR is based on an un-merged PR #650 , so there will be some commits that are no belong to this PR, please follow this link to see the actual patch: https://github.com/pingcap/tidb-operator/pull/658/files/f07c30c58aa83b85d3aee167f84884d704085755

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
  - [x] docs-cn: https://github.com/pingcap/docs-cn/pull/1630
  - [x] docs: https://github.com/pingcap/docs/pull/1347

### Does this PR introduce a user-facing change?:
 
 ```release-note
Support multiple tidb clusters management in Alibaba Cloud.
 ```

### CODE REVIEW

**Comment by tennix** [comment]
Please resolve the conflict.

**Comment by aylei** [comment]
@tennix done

**Comment by aylei** [comment]
@tennix @onlymellb I've addressed some merge conflicts, could you please take a look again?

File: deploy/modules/aliyun/tidb-cluster/variable.tf [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+variable "tidb_version" {
+  description = "TiDB cluster version"
+  default     = "v3.0.0"
+}
+
+variable "tidb_cluster_chart_version" {
+  description = "tidb-cluster chart version"
+  default     = "v1.0.0-beta.3"
```
Comment (weekface) [inline deploy/modules/aliyun/tidb-cluster/variable.tf]: use `rc.1`?

**Pull request merged**